### DISCUSSION TITLE
What actually changed in v99.3 Release from v99.1

### DISCUSSION THREAD

**Discussion by gutley** [mozilla-mobile/firefox-ios#10619] [General]
Looking at the commit history for the v99.3 branch the only thing that seems to have changed is the version number itself.

Is there some other magic the build process adds for you guys that isn't in the public repo?

**Comment by lmarceau** [comment]
@gutley you noticed well, build is indeed the same. It was made to be able to update App Store content.

**Reply by lmarceau** [reply]
@gutley just made the [release](https://github.com/mozilla-mobile/firefox-ios/releases/tag/v99.3) in Github as it wasn't done yet, thanks for raising this!

**Reply by gutley** [reply]
Thanks - Good 'ol App Store restrictions 😀
